client/post-editor/editor-ground-control/style.scss

```diff
@@ -49,7 +49,6 @@
 	display: flex;
```

Comment:
Thanks for giving this a go!
Adding a class was something that crossed my mind, though it'd have had to be something more generic than `is-saving` since there are a few possible display states, something like: `is-able-to-save`, `is-saving`, `is-saved` so the class might have had to be `is-visible` or something, which isn't a terrible option really.

I think before I merge, I'll give this approach a better run-through - like you said, could be a smaller (neater) diff footprint, but on top of that it saves popping a component in and out of memory.